packages/patternfly-4/react-core/src/components/Tooltip/Tooltip.js

```diff
@@ -21,6 +21,8 @@ export const TooltipPosition = {
 const propTypes = {
   /** Tooltip position */
   position: PropTypes.oneOf(Object.keys(TooltipPosition).map(key => TooltipPosition[key])),
+  /** Tooltip trigger: click, mouseenter, focus and etc */
```

Comment:
I would say let's be specific here and limit it to just 3 of the possibly 4 tippy variants (cannot support `manual` at the moment since we don't expose the isVisible prop for controlled mode)
```suggestion
  /** Tooltip trigger: click, mouseenter, focus */
```